### PR TITLE
Tooltip does not open on mobile

### PR DESCRIPTION
**Pull request by kirtijadhav**
see issue: #891 

### Visible/Frontend Changes
- [x] fix child observation tooltip click

### Architectural/Backend Changes
- [ ] 
- [ ]

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "ndb-core",
```
Comment (TheSlimvReal) [inline package-lock.json:8]: Can you please undo these changes.

File: src/app/core/entity-components/entity-form/entity-form/entity-form.component.html [hunk 2]
```diff
@@ -18,6 +18,13 @@
           }"
         >
         </ng-template>
+        <mat-icon
+          (click)="tooltipElement.show()" #tooltipElement="matTooltip"
```
Comment (TheSlimvReal) [inline src/app/core/entity-components/entity-form/entity-form/entity-form.component.html]:
```suggestion
          #tooltipElement="matTooltip"
          (click)="tooltipElement.show()"
```

**Review by TheSlimvReal — CHANGES_REQUESTED** [review]
Looks good so far, please also remove the tooltip from the following components:

- [ ] `EditLongText`
- [ ] `EditText`

Besides that I think this is mostly done.

File: package-lock.json (hunk 1 above)
Comment (kirtijadhav) [reply package-lock.json:8]: Yes I can. Thanks

Comment (kirtijadhav) [reply package-lock.json:8]: When I remove the tooltip component from EditLongText and EditText, position of the tooltip changes.
Please have a look on the attached image -

tooltip-error

Comment (TheSlimvReal) [reply package-lock.json:8]: Yes it looks like the `<ng-template>` takes up to much space. You should add some css rules so all the form elements have the same length and the tooltip icon does not go into the next line.

Comment (kirtijadhav) [reply package-lock.json:8]: Okay I will do that!

**Review by TheSlimvReal — CHANGES_REQUESTED** [review]
On some screen sizes the tooltip icon is still in a new line 
tooltip_displayed

**Comment by kirtijadhav** [comment]
> On some screen sizes the tooltip icon is still in a new line
> tooltip_displayed

Okay I will try to modify it!

**Review by TheSlimvReal — APPROVED** [review]
Works perfect now, good job!
